README.md

```diff
@@ -61,10 +61,17 @@ Make sure to install the Hugo extended version specified by the `HUGO_VERSION` e
 To build and test the site locally, run:
 
 ```bash
-# install dependencies
-npm ci
+# For MacOS, Linux etc.
```

Comment:
```suggestion
- For MacOS and Linux
```
This can be moved out of the code block and give a look like:

- For MacOS and Linux
  
  Install dependencies and deploy locally:
  ```bash
  npm ci
  make serve
  ```
- For Windows (PowerShell)

  Install dependencies and deploy locally:

  ```bash
  npm ci
  hugo.exe server --buildFuture --environment development
  ```